bin/run-tests-on-swarm.sh

```diff
@@ -226,7 +226,7 @@ fi
 
 VESPAVERSION=$(docker run --rm $BINDMOUNT_OPTS --entrypoint bash $DOCKERIMAGE -lc '${VESPA_HOME-/opt/vespa}/bin/vespa-print-default version')
 case "$VESPAVERSION" in
-    7.*.0) VESPAVERSION=7-SNAPSHOT
+    8.*.0) VESPAVERSION=8-SNAPSHOT
```

Comment:
Just leave both cases there so it works for 7 and 8 in a limited time period.

Reply:
Added 7.*.0 back